crates/meroctl/src/cli/context/alias.rs

```diff
@@ -51,14 +54,38 @@ impl ContextAliasCommand {
         let multiaddr = fetch_multiaddr(&config)?;
 
         match self.command {
-            ContextAliasSubcommand::Add { alias, context_id } => {
+            ContextAliasSubcommand::Add {
+                alias,
+                context_id,
+                force,
+            } => {
                 if !context_exists(&multiaddr, &config.identity, &context_id).await? {
                     environment.output.write(&ErrorLine(&format!(
                         "Context with ID '{}' does not exist",
                         context_id
                     )));
                     return Ok(());
                 }
+
+                let lookup_result = lookup_alias(multiaddr, &config.identity, alias, None).await?;
+                if let Some(existing_context) = lookup_result.data.value {
+                    if existing_context != context_id {
+                        if !force {
+                            environment.output.write(&ErrorLine(&format!(
+                            "Alias '{}' already exists and points to '{}'. Use --force to overwrite.",
+                            alias,
+                            existing_context
+                        )));
+                            return Ok(());
+                        }
+
+                        println!(
+                            "Warning: Overwriting existing alias '{}' from '{}' to '{}'",
+                            alias, existing_context, context_id
+                        );
+                    }
```

Comment:
let's make this easier to read, simplifying the overhead of thinking through the conditions

and since you still haven't removed the warnings, we might as well standardize them, so you can introduce a warn/warning equivalent of `ErrorLine` / `InfoLine`

```suggestion
                    if existing_context == context_id {
                        environment.output.write(&WarnLine(&format!(
                            "Alias '{alias}' already points to '{context_id}'. Doing nothing."
                        )));

                        return Ok(());
                    }

                    if !force {
                        environment.output.write(&ErrorLine(&format!(
                            "Alias '{alias}' already exists and points to '{existing_context}'. Use --force to overwrite."
                        )));

                        return Ok(());
                    }


                    environment.output.write(&WarnLine(&format!(
                        "Overwriting existing alias '{alias}' from '{existing_context}' to '{context_id}'"
                    )));
```

let's apply this everywhere else